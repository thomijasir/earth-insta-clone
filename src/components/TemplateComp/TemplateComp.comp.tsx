import React, { FC } from 'react';
import './TemplateComp.scss';

export type ITemplateCompProps = {};

export const TemplateCompDefaultProps = {};

export const TemplateCompNamespace = 'TemplateComp';

const TemplateComp: FC<ITemplateCompProps> = () => {
  // Component TemplateComp
  return (
    <div className="template-comp">
      <div className="comp">Area Component</div>
    </div>
  );
};

TemplateComp.displayName = TemplateCompNamespace;
TemplateComp.defaultProps = TemplateCompDefaultProps;
export default React.memo(TemplateComp);
