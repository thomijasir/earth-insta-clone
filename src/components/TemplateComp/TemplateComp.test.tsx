import React from 'react';
import render from 'react-test-renderer';
import TemplateComp from './TemplateComp.comp';

afterEach(() => {
  jest.clearAllMocks();
});

describe('Loading General Component Snap Test', () => {
  describe('render()', () => {
    test('renders the component', () => {
      const wrapper = render.create(<TemplateComp />);
      expect(wrapper.toJSON()).toMatchSnapshot();
    });
  });
});
