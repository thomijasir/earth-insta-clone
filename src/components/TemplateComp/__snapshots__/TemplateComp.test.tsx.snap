// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading General Component Snap Test render() renders the component 1`] = `
<div
  className="template-comp"
>
  <div
    className="comp"
  >
    Area Component
  </div>
</div>
`;
